.github/actions/rpm-delivery/action.yml

```
@@ -96,10 +111,16 @@ runs:
           scp -o StrictHostKeyChecking=no "$FILE" "${{ inputs.yum_repo_address }}:$TARGET" 2>&-
         done
 
+        # Cleanup is done on unstable repository only
+        if [ "${{ inputs.stability }}" == unstable ]; then
+          ssh -o StrictHostKeyChecking=no "${{ inputs.yum_repo_address }}" "ls -drc $PROJECT_LOCATION/* 2>&- | head -n -6 | xargs rm -rf"
```

as discussed, we should keep only last directory
```suggestion
          ssh -o StrictHostKeyChecking=no "${{ inputs.yum_repo_address }}" "ls -drc $PROJECT_LOCATION/* 2>&- | head -n -1 | xargs rm -rf"
```